starter-code/index.html

```diff
@@ -1,31 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Instagram</title>
-</head>
-<body>
-  Instagram
+  <head>
+    <title>Instagram</title>
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
 
-  Capture and Share the World's Moments
+  <body>
+    <section>
+      <!--IMAGE IPHONE-->
```

Comment:
Thanks for all your comments. Really readable code and so I really appreciate it :)